src/ensembl/src/content/home/components/homepage-app-links/HomepageAppLinks.tsx

```diff
@@ -0,0 +1,182 @@
+import React, { useState, useEffect, useRef } from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import classNames from 'classnames';
+
+import { isEnvironment, Environment } from 'src/shared/helpers/environment';
+import * as urlFor from 'src/shared/helpers/urlHelper';
+import useOutsideClick from 'src/shared/hooks/useOutsideClick';
+
+import { fetchDataForLastVisitedObjects } from 'src/content/app/browser/browserActions';
+
+import { getEnabledCommittedSpecies } from 'src/content/app/species-selector/state/speciesSelectorSelectors';
+import {
+  getPreviouslyViewedGenomeBrowserObjects,
+  PreviouslyViewedGenomeBrowserObjects
+} from 'src/content/home/homePageSelectors';
+
+import ImageButton from 'src/shared/components/image-button/ImageButton';
+
+import { ReactComponent as BrowserIcon } from 'static/img/launchbar/browser.svg';
+import { ReactComponent as EntityViewerIcon } from 'static/img/launchbar/entity-viewer.svg';
+import { ReactComponent as HomeIcon } from 'static/img/header/home.svg';
+
+import styles from './HomepageAppLinks.scss';
+
+import { RootState } from 'src/store';
+import { Status } from 'src/shared/types/status';
+import { CommittedItem } from 'src/content/app/species-selector/types/species-search';
+
+type Props = {
+  species: CommittedItem[];
+  previouslyViewedGenomeBrowserObjects: PreviouslyViewedGenomeBrowserObjects;
```

Comment:
This will likely not survive anyway. The links we are showing on the home page are currently conditional on whether the user has viewed anything in genome browser; the updated specifications say that we just show a list of selected species regardless of whether user has viewed anything in genome browser.
I suggest we don't touch this code, and when EntityViewer is ready, simply delete it.

Reply:
Let's leave it for now then